### PR TITLE
F-12 added rating & list

### PR DESCRIPTION
**Pull request by sushmita12345**
Added Rating and Lists component to the Component Library.

Screenshot 2022-03-25 at 8 58 23 PM

Screenshot 2022-03-25 at 9 11 15 PM

Screenshot 2022-03-25 at 9 11 26 PM

### CODE REVIEW

File: assets/css/list.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+.list-unordered-container {
+  display: flex;
+  justify-content: space-between;
+}
+
+ul {
+  font-size: 1.5rem;
+}
```
Comment (vaishnavipaliya11) [inline assets/css/list.css:35]: as a suggestion you should style directly to html. Instead  you can give class or id.

**Review by vaishnavipaliya11 — APPROVED** [review]

**Comment by vaishnavipaliya11** [comment]
The code looks clean and appropriate.

**Review by sakshamAK — APPROVED** [review]
You're good to go👍

**Comment by sushmita12345** [comment]
Thank you

**Pull request merged**